polus-infering-splinedist-plugin/README.md

```diff
@@ -0,0 +1,47 @@
+# WIPP Widget
```

Comment:
This README looks like it might have been copy and pasted from the splinedist training plugin, but only partially modified. Please review the entire thing and edit accordingly.